Makefile

```
@@ -89,14 +89,22 @@ bootstrap: clear-dbs ## Bootstrap the node from scratch
 		--chain-dir $(CHAIN_DIR) \
 		--network $(NETWORK)
 
-dev: ## Compile and run for development with default options
+dev: start # 'backward-compatibility'; might remove after a while.
```

checkmake (0.2.2)

[warning] Target "dev" should be declared PHONY.

(phonydeclared)